MEL/MEL/PressureLayer.cs

```diff
@@ -41,13 +41,6 @@ public PressureLayer(string name)
 		/// </summary>
 		public void Add(RasterizedLayer rasterizedLayer, float influence)
 		{
-			if (!rasterizedLayer.IsLoadedCorrectly)
```

Comment:
@TeraNaidja Ha Phil, note that `RasterizeLayers` in PressureLayers.cs has this check, see:
```
		public void RasterizeLayers(MEL mel)
		{
			if (redraw)
			{
				//double total = 0f;

				rawData = new double[MEL.x_res, MEL.y_res];
				Console.WriteLine("rasterizing " + name);
				redraw = false;

				if (mel.ApiConnector.ShouldRasterizeLayers)
				{
					try
					{
						foreach (LayerEntry layerEntry in layers)
						{
							if (layerEntry == null)
							{
								Console.WriteLine("null layer");
								continue;
							}

							if (!layerEntry.RasterizedLayer.IsLoadedCorrectly)
							{
								Console.WriteLine(
									$"Tried to rasterize {layerEntry.RasterizedLayer.name}, but the layer failed to load correctly");
								continue;
							}
```